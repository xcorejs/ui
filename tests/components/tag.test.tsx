import 'jest-styled-components';

import { Tag } from '@xcorejs/ui';
import React from 'react';
import renderer from 'react-test-renderer';

test('Tag component', () => {
  const component = renderer.create(
    <Tag>Tag</Tag>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
