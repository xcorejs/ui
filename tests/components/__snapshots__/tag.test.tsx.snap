// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag component 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  white-space: nowrap;
  border: 1px solid #455663;
  border-radius: 0.3rem;
  padding-left: 0.8rem;
  padding-right: 0.8rem;
  font-size: 1.2rem;
  font-weight: 500;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  line-height: 2rem;
  font-weight: 500;
  font-size: 1.2rem;
  font-family: "rubik",-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-transition: color 300ms,background 300ms,border-color 300ms;
  transition: color 300ms,background 300ms,border-color 300ms;
  color: #fff;
  background-color: #455663;
}

<div
  className="c0"
  color="background"
  fontFamily="text"
  fontSize="1.2rem"
  fontWeight={500}
>
  Tag
</div>
`;
